### DISCUSSION TITLE
Beta updates after v15.0.0

### DISCUSSION THREAD

**Discussion by tekezo** [pqrs-org/Karabiner-Elements#3900] [Announcements]
# v15.0.1

-   ✨ New Features
    -   A setting has been added to disable mouse cursor movement and scrolling for both the mouse and gamepad.

**Comment by tekezo** [comment]
# v15.0.2

- Settings of the gamepad stick have been added; deadzone and delta magnitude detection threshold.
- Change delta_magnitude_detection_threshold default value (0.01 -> 0.02)
- The appendix/GamePadViewer was specialized for displaying the status of the sticks, and the Converter was removed.

**Comment by tekezo** [comment]
# v15.0.3

- Update information layout in GamePadViewer
- Add quitUsingKeyboardShortcut into EventViewer.UserSettings

**Comment by tekezo** [comment]
# v15.0.4

-   💥 Breaking changes
    -   In previous versions, when fn+arrows were implicitly changed to home, end, page up, and page down, this would occur even if other modifiers were pressed.
        For example, fn+shift+up_arrow would become shift+page_up.
        This has now been changed so that home, end, page up, and page down are only triggered when only fn is pressed.
        If you prefer the old behavior, you can revert to it by disabling the "Change fn+arrows..." setting in the Virtual Keyboard tab.
        <br/><br/>
        <table><tbody><tr><td>
        <img src="https://karabiner-elements.pqrs.org/docs/releasenotes/images/v15.1.0/strict-fn-arrows@2x.png" alt="Karabiner-Elements Setttings" />
        </td></tr></tbody></table>

**Comment by tekezo** [comment]
# v15.0.14

- The implicit conversion feature for fn+arrow keys, fn+return, and fn+delete has been removed.
   For example, fn+up arrow used to be automatically changed to page up, but it will now be treated as fn+up arrow without modification.
-   To improve the behavior around the fn key, the Vendor ID and Product ID of the virtual keyboard used by Karabiner-Elements have been changed to the same values as those of a real Apple external keyboard.
-   Remove country_code, strict_fn_arrows from virtual_hid_keyboard, add keyboard_type_v2.
-   Remove country_code support from configuration_json_helper
-   Remove operation_type::set_keyboard_type
-   Add killall Karabiner-VirtualHIDDevice-Daemon to postinstall
-   Remove connected_devices::details::device, connected_devices uses device_properties instead.

**Comment by tekezo** [comment]
# v15.0.15

-   Updated the embedded screenshots and help messages for macOS 15 Sequoia.

**Comment by tekezo** [comment]
# v15.0.16

- Made the game_pad_stick_converter active only for gamepad devicdes, and the timers and other settings are reset when the device is not seized.
- Use overlay instead of bold for hoverRule in ComplexModificationsView.
- Use overlay instead of bold for hoverRule in ProfilesView.
- Save by command+s in ComplexModificationsEditView.

**Comment by tekezo** [comment]
# v15.0.17

-   There were two JSON files that output information about connected devices, but this information has now been consolidated into `karabiner_grabber_devices.json`.
        `karabiner_grabber_device_details.json` is no longer used. Please use `karabiner_grabber_devices.json` from now on.
    -   `karabiner_grabber_devices.json`
    -   `karabiner_grabber_device_details.json` (obsoleted)

**Comment by tekezo** [comment]
# v15.0.18

-   Fixed an issue where disconnected devices would occasionally remain in the device list.

**Comment by tekezo** [comment]
# v15.0.19

-   Added [open_application](https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to/software_function/open_application/) to complex modifications.

**Comment by tekezo** [comment]
# v15.0.20

- Remove grabberClient.connected check in KeyboardTypeSelectorView.

**Comment by titust1** [comment]
> # v15.0.1
> * ✨ New Features
>   
>   * A setting has been added to disable mouse cursor movement and scrolling for both the mouse and gamepad.

Why? scrolling for the mouse has been disable.... what is the reason?
